components/Collections.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Button,
+  ButtonGroup,
+  Spinner,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { getServicesList } from 'api'
+import { ServicesList } from 'models'
+import { UserDoc } from 'models/users'
+import React, { useEffect, useState } from 'react'
+import CollectionItem from './CollectionItem'
+
+interface CollectionProps {
+  userData: UserDoc
+}
+
+export const Collections = ({ userData }: CollectionProps): JSX.Element => {
+  const [loading, setLoading] = useState(true)
+  const [listData, setListData] = useState<ServicesList[]>([])
+
+  useEffect(() => {
+    const promises = []
+    for (const list of userData.lists) {
+      const listData = getServicesList(list)
+      promises.push(listData)
+    }
+
+    Promise.all(promises.map((p) => p.catch((e) => e))).then((list) => {
+      setListData(list)
+      setLoading(false)
+    })
+  }, [userData.lists])
```

Comment:
This is done like this so that if ``userData.lists`` updates we will fetch the correct lists on the spot and update the UI.